transformers/component_tests/queries/queries_suite_test.go

```diff
@@ -1,6 +1,8 @@
 package queries
 
 import (
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 	"io/ioutil"
 	"testing"
 
```

Comment:
Can you remove spaces between imports in this file and then run `goimports -w transformers/component_tests/queries/queries_suite_test.go`